### PR TITLE
chore(ui): converting one screen to typescript

### PR DESCRIPTION
**Pull request by sarahscott**
starting with `Org.svelte` & its children

### CODE REVIEW

File: ui/DesignSystem/Component/SidebarLayout.svelte [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
   import Sidebar from "./Sidebar.svelte";
 
-  export let dataCy = null;
-  export let style = null;
+  export let dataCy = "";
+  export let style = "";
```
Comment (FintanH) [inline ui/DesignSystem/Component/SidebarLayout.svelte:12]: I'm curious as to what this setup is for? Just for my self learning :)

File: ui/Screen/Org.svelte [hunk 2]
```diff
@@ -30,14 +30,15 @@
...
+  import MenuItem from "../DesignSystem/Component/HorizontalMenu/MenuItem.svelte";
 
   const menuRoutes = {
     "/orgs/:id/": ProjectsMenu,
     "/orgs/:id/projects": ProjectsMenu,
     "/orgs/:id/members": MembersMenu,
   };
 
-  const topbarMenuItems = orgId => [
+  const topbarMenuItems = (orgId: string) => [
```
Comment (FintanH) [inline ui/Screen/Org.svelte:41]: Should `orgId` be a type `OrgId` to disambiguate it from other `string`s?

File: ui/src/identity.ts [hunk 3]
```diff
@@ -19,6 +19,7 @@ export interface Identity {
   };
   registered?: string;
   avatarFallback: Avatar;
+  shareableEntityIdentifier?: string;
```
Comment (FintanH) [inline ui/src/identity.ts:22]: I'd also consider pushing out the use of `string` everywhere and introducing types that have a better structure and/or allow us to disambiguate them from other `string`ly typed objects.

Comment (sarahscott) [reply ui/src/identity.ts:22]: agreed

File: ui/DesignSystem/Component/SidebarLayout.svelte (hunk 1 above)
Comment (sarahscott) [reply ui/DesignSystem/Component/SidebarLayout.svelte:12]: Figuring out the vibe for type inference v type declaration in `props`; still under construction 🚧

File: ui/Screen/Org.svelte (hunk 2 above)
Comment (sarahscott) [reply ui/Screen/Org.svelte:41]: ja i think that would make sense for all of the `id`s

Comment (FintanH) [reply ui/Screen/Org.svelte:41]: Sehr gut

File: ui/src/identity.ts (hunk 3 above)
Comment (sarahscott) [reply ui/src/identity.ts:22]: This might get a little awkward for components that render `string`s though, like `Title` or `Text`

Comment (FintanH) [reply ui/src/identity.ts:22]: Oh ya, sure, They're literally strings. No argument there :)

File: ui/DesignSystem/Primitive/Icon/Verified.svelte [hunk 4]
```diff
@@ -1,8 +1,8 @@
 <script>
   import IconWrapper from "./IconWrapper.svelte";
 
-  export let dataCy = null;
-  export let style = null;
+  export let dataCy = "";
+  export let style = "";
```
Comment (xla) [inline ui/DesignSystem/Primitive/Icon/Verified.svelte:5]: There is some work around required/optional props in sveltejs/svelte#5183, sveltejs/language-tools#273 and sveltejs/language-tools#319.

File: ui/Screen/Org.svelte [hunk 5]
```diff
@@ -58,7 +59,24 @@
     },
   ];
 
-  let registerProjectMenuItem;
+  interface MenuItem {
```
Comment (xla) [inline ui/Screen/Org.svelte:62]: Would be nice to push these into `src` code.

**Comment by sarahscott** [comment]
Closing until we figure out the linting situation (#779 )

**Pull request closed without merging**